docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
 
   packet-forwarder:
     image: nebraltd/hm-pktfwd:1213210
```

Comment:
Is there a reason we are using this instead of nebraltd/hm-pktfwd:57f52a5?

Reply:
FYI the latest GA rolled back the changes to the regions so think we can move back to master on packet forwarder

Reply:
Yea both good points. I did mean for this to be `nebraltd/hm-pktfwd:57f52a5`. But that does raise the issue of [region changes not being in master yet](https://github.com/NebraLtd/hm-pktfwd/pull/69). I just approved that PR and its probably safest to merge that and then point to that hash, post-merge.

Reply:
We don't need the region changes anymore though as Helium reverted that

Reply:
You don't think they might just pop back up again suddenly?

Reply:
I mean, I would hope not. But I suppose it doesn't cause any harm to have both

Reply:
`nebraltd/hm-pktfwd:3f7c41e` 👍

Reply:
Done: https://github.com/NebraLtd/helium-miner-software/pull/215/commits/504a3e7697540d7be0671dc434f369c14f9c2ce7#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R28